L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker_cfg.py

```diff
@@ -235,9 +227,15 @@
 process.ntuple = cms.Path(process.L1TrackNtuple)
 
 process.out = cms.OutputModule( "PoolOutputModule",
-                                fastCloning = cms.untracked.bool( False ),
+     #  #                         fastCloning = cms.untracked.bool( False ),
+                                outputCommands = process.RAWSIMEventContent.outputCommands,
+ #                               dataset = cms.untracked.PSet(
+    #                                filterName = cms.untracked.string(''),
+  #                                  dataTier = cms.untracked.string('GEN-SIM')
+   #                             ),
```

Comment:
Are the commented lines necessary? Can they be removed?

Reply:
done